lib/rules/no-unmodified-loop-condition.js

```diff
@@ -165,6 +165,8 @@ function updateModifiedFlag(conditions, modifiers) {
 
 module.exports = {
     meta: {
+        type: "suggestion",
```

Comment:
I think this could be `problem` since the goal is to catch bugs, although there are also some cases where it reports false positives.

Reply:
Agreed.